cmd/cloudant_exporter/main.go

```diff
@@ -133,6 +134,19 @@ type monitorLooper struct {
 func (rc *monitorLooper) Go() {
 	ticker := time.NewTicker(rc.Interval)
```

Comment:
I think we need to move the ticker to _after_ the Sleep, because otherwise the ticks are not affected by the perturbation (that is, the ticker tries to tick every `Interval` from when it is created, _not_ at `Interval` after the last tick completed).

Reply:
👍